man/docker-daemon.8.md

```diff
@@ -0,0 +1,402 @@
+% DOCKER(1) Docker User Manuals
+
+% William Henry 
+
+% SEPTEMBER 2015
+
+# NAME
+docker-daemon - Enable daemon mode
```

Comment:
This (`Enable daemon mode`) is inline with what `docker daemon --help` displays. Let's wait for other maintainers to comment.